### PR TITLE
test: Add tests for panel permissions

### LINKED ISSUE #5467: Adding dredd tests for panel permission
**Is your feature request related to a problem? Please describe.**

Add dredd tests for panel persmission
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

I'm working on it!

### PR DESCRIPTION
**Pull request by prabhupant**
Fixes #5467 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
adds tests for panel permissions

#### Changes proposed in this pull request:

- add dredd hooks for panel permissions

### CODE REVIEW

**Comment by prabhupant** [comment]
@iamareebjamal @srv-twry please review

File: tests/hook_main.py [hunk 1]
```diff
@@ -4241,3 +4241,85 @@ def custom_system_roles_panel_permission(transaction):
...
+    :return:
+    """
+    with stash['app'].app_context():
+        panel_permission = PanelPermissionFactory()
+        db.session.add(panel_permission)
+        db.session.commit()
+
+
+@hooks.before("Panel Permissions > Panel Permission Details for a Custom System Role >"
+                " Panel Permission Details for a Custom System Role")
```
[the remark this answers is not in the document]
Comment (srv-twry) [reply tests/hook_main.py]: Please fix this one.

**Comment by prabhupant** [comment]
@srv-twry i don't know why the travis build has failed. I just corrected fixed the hound violation

**Comment by srv-twry** [comment]
@prabhupant Check the Travis build. Two unit tests are failing.

**Comment by prabhupant** [comment]
@srv-twry this link in `image_url_test = 'https://cdn.pixabay.com/photo/2014/09/08/17/08/hot-air-balloons-439331_960_720.jpg'` in the file `tests/api/helpers/test_file.py` was not working. I have changed it now and the build has passed. Please review the PR

**Pull request merged**